website/content/docs/oss/installing/run-and-login.mdx

```diff
@@ -17,16 +17,17 @@ $ boundary dev
 
 Boundary uses a predictable login name (`admin`) and password (`password`) in
 dev mode. These can be overridden, or randomly generated, with flags to
-`boundary dev`.
+`boundary dev`. The auth method resource resides in the global scope and is set
+as the primary for that scope, which means you don't need to specify the
+auth method ID as a flag when using the authenticate command, all you need
+is the short form `boundary authenticate`.
 
 <Tabs>
 <Tab heading="CLI">
 
 
 ```
-$ boundary authenticate password \
-         -login-name=admin \
-         -auth-method-id=ampw_1234567890
+$ boundary authenticate 
```

Comment:
same, I'd prefer to see output here

```suggestion
$ boundary authenticate
Please enter the login name (it will be hidden):
Please enter the password (it will be hidden):
```